patternfly/patternfly-elements
# fix: issue with slots containing dashes in JS file

### What has changed and why
Summarize files edited as part of this MR along with a brief description of what was changed/why.

* Update generator JS file to convert slot names to camel case

### Testing instructions
Be sure to include detailed instructions on how your update can be tested by another developer.

1. Run the generator using `npm run new`
2. Add at least 2 slots, `message-on, color` (one with a dash and one without)
3. Check the generated element to ensure the JS has converted the slot name with a dash in it into camel case

Your repository infrastructure updates should work for at least:

### Ready-for-merge Checklist



**Be sure to share your updates with the [patternfly-elements-contribute@redhat.com](mailto:patternfly-elements-contribute@redhat.com) mailing list!**
